lib/DDG/Spice/Emojipedia.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Emojipedia;
+
+# ABSTRACT: Provide definitions and meanings for emoji characters.
+
+use strict;
+use utf8;
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://api.emojipedia.org/emojis/$1?api_key={{ENV{DDG_SPICE_EMOJIPEDIA_APIKEY}}}&format=json';
```

Comment:
@mathewbyrne good news, I think we figured it out!

It looks like the API call redirects from this adding a `/` after the `$1`! Our NGINX doesn't follow that redirect which is why we're not getting a response back!

Can you please change this to `http://api.emojipedia.org/emojis/$1/?` 

We'll redeploy once that is done and then we can confirm if that's been causing the problems!